modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -467,6 +479,13 @@ export default {
     .apos-area-widget-inner &.apos-highlight:before {
       z-index: $z-index-default;
     }
+
+    &.apos-foreign {
+      outline: 1px dashed var(--a-warning);
+      /deep/ .apos-button {
+        background-color: var(--a-warning);
```

Comment:
Let's also make the color of the icon white to match the breadcrumbs. `var(--a-white)`

Reply:
This still needs the white icon. That should be done in aposWidgetControls, passing the `fill-color` prop into the icon.

Reply:
Though setting `color: var(--a-white)` here seems to be enough.

Reply:
I'm talking about this:
![Screen Shot 2020-11-23 at 9 22 48 AM](https://user-images.githubusercontent.com/1155984/99980090-76a2d480-2d6d-11eb-9a62-6d526c842467.png)

The pencil on yellow not matching the text on yellow.